Gruntfile.js

```diff
@@ -357,7 +349,6 @@ module.exports = function(grunt) {
     }
   });
 
+  grunt.registerTask('test', ['mochaTest'/*, 'casperjs'*/]);
```

Comment:
Should we update the package.json's test command as it curently does make test not grunt test. Im not sure how much it matters really at Can we all use php storm's mocha test runner to run unit tests.